src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs

```diff
@@ -24,19 +24,19 @@ public class XbmcMetadata : MetadataBase<XbmcMetadataSettings>
     {
         private readonly Logger _logger;
         private readonly IMapCoversToLocal _mediaCoverService;
-        private readonly ITagService _tagService;
+        private readonly ITagRepository _tagRepo;
```

Comment:
Had to switch this direct to repo to avoid a recursive dependency loop (outlined below)

![image](https://github.com/Sonarr/Sonarr/assets/376117/7a739d9c-4194-46c0-a472-5b466ed37456)